lib/api.js

```diff
@@ -61,7 +62,9 @@ module.exports = settings => {
   flow.hook('pre-create', hooks.unique(settings));
 
   flow.hook('create', hooks.meta(settings));
+  flow.hook('create', hooks.updateAction(settings));
```

Comment:
This is surely unnecessary by definition? If we're creating a new task then it can't be necessary to also update an existing task?

Reply:
I'm wrong here I think, right?

This does need to be here because the task is _always_ a `type: 'grant'` until it is modified here?

Reply:
yes